src/Microsoft.OpenApi.OData.Reader/Operation/EntitySetPostOperationHandler.cs

```diff
@@ -104,23 +125,43 @@ protected override void SetResponses(OpenApiOperation operation)
                 };
             }
 
+            IDictionary<string, OpenApiMediaType> content;
+
+            if (EntitySet.EntityType().HasStream)
+            {
```

Comment:
extract this common logic to a method so you don't have to repeat it.

Reply:
Resolved